cpp/src/c_api/graph_sg.cpp

```diff
@@ -160,6 +181,40 @@ struct create_graph_functor : public cugraph::c_api::abstract_functor {
         cugraph::graph_view_t<vertex_t, edge_t, store_transposed, multi_gpu>,
         edge_type_id_t>(handle_);
 
+      if (drop_multi_edges_) {
+        std::cout << "calling drop_multi_edges" << std::endl;
+        raft::print_device_vector(
+          "  edgelist_srcs", edgelist_srcs.data(), edgelist_srcs.size(), std::cout);
+        raft::print_device_vector(
+          "  edgelist_dsts", edgelist_dsts.data(), edgelist_dsts.size(), std::cout);
+
+        std::tie(
+          edgelist_srcs, edgelist_dsts, edgelist_weights, edgelist_edge_ids, edgelist_edge_types) =
+          cugraph::sort_and_remove_multi_edges(handle_,
+                                               std::move(edgelist_srcs),
+                                               std::move(edgelist_dsts),
+                                               std::move(edgelist_weights),
+                                               std::move(edgelist_edge_ids),
+                                               std::move(edgelist_edge_types));
+
+        std::cout << "after drop_multi_edges" << std::endl;
+        raft::print_device_vector(
+          "  edgelist_srcs", edgelist_srcs.data(), edgelist_srcs.size(), std::cout);
+        raft::print_device_vector(
+          "  edgelist_dsts", edgelist_dsts.data(), edgelist_dsts.size(), std::cout);
+      }
+
+      if (drop_self_loops_) {
+        std::tie(
+          edgelist_srcs, edgelist_dsts, edgelist_weights, edgelist_edge_ids, edgelist_edge_types) =
+          cugraph::remove_self_loops(handle_,
+                                     std::move(edgelist_srcs),
+                                     std::move(edgelist_dsts),
+                                     std::move(edgelist_weights),
+                                     std::move(edgelist_edge_ids),
+                                     std::move(edgelist_edge_types));
+      }
```

Comment:
Sorry for nitpicking but if `drop_self_loops` appear before `drop_multi_edges` in the input parameters, should we better perform drop_self_loops() before drop_multi_edges()? This consistency may make the code easier to read and slightly easier to maintain.

Reply:
Will address in next push